src/main/java/net/pms/store/StoreItem.java

```diff
@@ -442,6 +443,20 @@ public TranscodingSettings resolveTranscodingSettings() {
 			return null;
 		}
 
+		// WebStreams should be checked against the mime-type of the stream
+		if (this instanceof WebStream ws) {
+			if (getMediaInfo() != null && renderer.getFormatConfiguration() != null) {
+				if (StringUtils.isAllBlank(getMediaInfo().getMimeType())) {
+					LOGGER.warn("cannot read MIME-TYPE of stream {} ...", ws.getUrl());
```

Comment:
Should this line also return early?

Reply:
If we cannot read the mime-type of the web-stream, we may need transcoding. Therefore, this line shouldn't return early. I haven't seen any web-streams without mine-type set, yet ...